integrations/oci/sdk/cdi/src/main/java/module-info.java

```diff
@@ -40,4 +40,6 @@
 
     provides javax.enterprise.inject.spi.Extension
         with io.helidon.integrations.oci.sdk.cdi.OciExtension;
+
+    opens io.helidon.integrations.oci.sdk.cdi to weld.core.impl;
```

Comment:
Oh interesting; does Weld actually reflect on the _extension_? I guess it must if it is obliged to make a bean out of it?